src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/CompletionResolveHandler.cs

```diff
@@ -128,6 +129,16 @@ public async Task<CompletionItem> HandleRequestAsync(CompletionItem request, Cli
             var formattingOptions = _formattingOptionsProvider.GetOptions(requestContext.HostDocumentUri);
             if (resolvedCompletionItem.TextEdit != null)
             {
+                if (!formattingOptions.InsertSpaces)
```

Comment:
Do we need to go the opposite direction too? Aka if it's "insertSpaces" but the edit has tabs?